core/kernel/dt.c

```diff
@@ -51,6 +51,101 @@ bool dt_have_prop(const void *fdt, int offs, const char *propname)
 	return prop;
 }
 
+int dt_get_irq(void *fdt, int node)
+{
+	const uint32_t *int_prop = NULL;
+	int len_prop = 0;
+	int it_num = -1;
```

Comment:
suggest to introduce `DT_INFO_INVALID_INTERRUPT` as done for `DT_INFO_INVALID_REG`.

Reply:
ok